runtime/doc/lsp.txt

```diff
@@ -1604,6 +1587,53 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (`integer`)
 
 
+==============================================================================
+Lua module: vim.lsp.completion                                *lsp-completion*
+
+*vim.lsp.completion.BufferOpts*
+
+    Fields: ~
+      • {autotrigger}?  (`boolean`) Whether to trigger completion
+                        automatically. Default: false
+
+
+disable({client_id}, {bufnr})                   *vim.lsp.completion.disable()*
+    Disables completions from the given language client in the given buffer.
+
+    Parameters: ~
+      • {client_id}  (`integer`) Client ID
+      • {bufnr}      (`integer`) Buffer handle, or 0 for the current buffer
+
+enable({client_id}, {bufnr}, {opts})             *vim.lsp.completion.enable()*
+    Enables completions from the given language client in the given buffer.
```

Comment:
disable() is redundant. enable(enable: boolean) is the conventional signature. this also avoids a potential "toggle()" function in the future. see `:help dev-pattens`